src/guards.js

```diff
@@ -68,7 +68,16 @@ export const isAskedForCurrentMods = (text) => {
  * @returns {boolean}
  */
 export const isAskedForCurrentWinners = (text) => {
-    return /^who/.test(text) && /winners|new mod|will win|future mod/.test(text);
+    return /^who/.test(text) && /winners|new mod|will win|future mod/i.test(text);
```

Comment:
Just checking if there's any reason for the case-insensitive flags since it's all in lowercase

Reply:
Just in case users start with `Who` :) I already noticed this several times (and at least once today) - I am pretty sure insensitive match won't hurt anything, but will improve user experience

Reply:
That's what this line is for, all text comparison is done in lowercase

[`const content = originalMessage.toLowerCase().replace(/^@\S+\s+/, '');`](https://github.com/samliew/se-electionbot/blob/master/src/index.js#L319)

Reply:
I forgot about this line, honestly... I remember the "mentioned" part but thought that the string is passed as is without lowercasing, my bad!

Reply:
I don't think it will affect performance so we can leave it